### ISSUE TITLE
fpack watch mode fails

### ISSUE THREAD

**Issue by tcoopman** [fastpack/fastpack#128]
fpack 0.6.0

```bash
node_modules/.bin/fpack watch src/index.js --preprocess='\\.png$:url-loader'
Packed in 0.021s. Bundle: 955Kb. Modules: 60.  Cache: used. Mode: development.
fpack: internal error, uncaught exception:
       FastpackUtil__Process.NotRunning("watchman --no-save-state -j --no-pretty -p")
       Raised at file "Fastpack/Watcher.re", line 162, characters 2-282
       Re-raised at file "Fastpack/Watcher.re", line 228, characters 2-1023
       Re-raised at file "Fastpack/Commands.re", line 90, characters 10-109
       Re-raised at file "src/core/lwt.ml", line 3008, characters 20-29
       Called from file "src/unix/lwt_main.ml", line 42, characters 8-18
       Called from file "Fastpack/Commands.re", line 13, characters 10-15
       Called from file "src/cmdliner_term.ml", line 27, characters 19-24
       Called from file "src/cmdliner.ml", line 27, characters 27-34
       Called from file "src/cmdliner.ml", line 106, characters 32-39
```

**Comment by zindel** [comment]
Yeah, the issue is a bit cryptic, but it means that watchman is not installed. 
https://facebook.github.io/watchman/

**Comment by tcoopman** [comment]
That's useful information!

**Comment by tcoopman** [comment]
Installing took a bit of time, but this fixes it indeed.
You can close this issue, but I'm inclined to keep it open because of the error message and documentation?

**Comment by zindel** [comment]
Yeah, let's keep it open for now. I definitely need to document dependencies better

**Comment by zindel** [comment]
@tcoopman This should be fixed in `fastpack@0.8.0`.
